todo_app/app.py

```diff
@@ -1,28 +1,26 @@
 from flask import Flask, render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, get_item, add_item, save_item, STATUSES
+from todo_app.data.trello_items import get_items, add_item, move_item
```

Comment:
Nice, the fact this is such a clean change here is a good sign that we've picked a nice interface! 😄